CONTRIBUTING.md

```diff
@@ -6,15 +6,17 @@ We appreciate your help!
 
 ## Table of Contents
 
-- [Contributing Issue](#Contributing-Issue)
-  - [Bug Report](#Bug-Report)
-  - [Proposal](#Proposal)
-  - [Feature Request](#Feature-Request)
-  - [Security Issue Report](#Security-Issue-Report)
-- [Contributing Source Code](#Contributing-Source-Code)
-  - [Before contributing source code](#Before-contributing-source-code)
-  - [How to contributing source code](#How-to-contributing-source-code)
-  - [Branch naming convention](#Branch-naming-convention)
+- [Contributing to Vald](#contributing-to-vald)
+  - [Table of Contents](#table-of-contents)
```

Comment:
I'm not sure why you added this line.
This section is `Table of Contents` 😅

Reply:
Sorry It is auto generated with any intention, I will update it.